### ISSUE TITLE
BLENDER 2.8 - problem with the transparency

### ISSUE THREAD

**Issue by iallo74** [KhronosGroup/glTF-Blender-IO#1057]
Hi everyone. I'm fairly new to Blender and I'm using it to export geometry for ThreeJS.
I have a strange problem with the transparency of 2 objects. The geometries are identical but are imported from 2 different .obj files. How come they behave like this? Even if I combine them with CTRL + J it's the same thing.

The files:
[https://www.scuoladimassaggio.com/ss3.jpg](https://www.scuoladimassaggio.com/ss3.jpg)
[https://www.scuoladimassaggio.com/ss3.blend1](https://www.scuoladimassaggio.com/ss3.blend1)

I thank you for any help.

**Comment by scurest** [comment]
Looks like alpha sorting issues.

![0](https://user-images.githubusercontent.com/11024420/81221845-fbccc480-8fa8-11ea-90f5-8d8d5e32c446.png)

In the material settings panel, switch the blend mode to "Opaque" or "Alpha Clip". Or turn on "Backface Culling".

(You can also try "Alpha Hashing" or turn off "Show Backface" but that won't change the exported glTF.)

**Comment by iallo74** [comment]
Thanks for the quick reply, scurest. The problem is that I have to export the object to glTF to use it in ThreeJS. I don't know how to get rid of that bad effect ...

**Comment by donmccurdy** [comment]
Could you attach the glTF file you are getting? I'm not sure if something is weird in the file or if my local installation is out of date, but I'm not getting any transparency when exporting this.

**Comment by iallo74** [comment]
RESOLVED!
The problem was caused by the ordering of the face indexes. I rearranged the faces manually: for each minimal piece of geometry, I positioned the cursor in the center (RBM menu -> Snap vertices -> Cursor to selected) and then I ordered the faces by "Cursor distance" (Mesh menu -> Sort elements -> Cursor distance -> Faces).

Thanks donmccurdy: transparency is not exported to the glTF file, but I manage it later with ThreeJS in my web application.
